titus-supplementary-component/job-activity-history/src/main/java/com/netflix/titus/supplementary/jobactivity/endpoint/grpc/JobActivityHistoryGrpcServer.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.supplementary.jobactivity.endpoint.grpc;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import com.netflix.titus.runtime.endpoint.common.grpc.AbstractTitusGrpcServer;
+import com.netflix.titus.runtime.endpoint.common.grpc.GrpcEndpointConfiguration;
+
+@Singleton
+public class JobActivityHistoryGrpcServer extends AbstractTitusGrpcServer {
```

Comment:
Other classes are using just `JobActivity`, not `JobActivityHistory`.

Reply:
Yes, `JobActivityHistory` is long, and I decided to use `JobActivity` name instead. I will fix it.